dashboard/app/models/levels/blockly.rb

```diff
@@ -268,6 +269,7 @@ def blockly_level_options
           default_toolbox_blocks
         level_prop['codeFunctions'] = try(:project_template_level).try(:code_functions) || code_functions
         level_prop['sharedBlocks'] = shared_blocks
+        level_prop['sharedFunctions'] = include_shared_functions && shared_functions
```

Comment:
Nit: `level_prop['sharedFunctions'] = shared_functions if include_shared_functions` so we don't get "false" when `!include_shared_functions`.